coverage/src/BamViewChart.js

```diff
@@ -396,10 +374,10 @@ function createBamView(bamHeader, data, element, bamViewControlsElement) {
                 svg.selectAll('.chromosome-label').remove();
 
                 // Re-draw the chromosome button for the selected chromosome
-                const chromosomes = svg.selectAll('.chromosome')
+                const chromosomes = svg.selectAll('.chromosome-span')
                     .data([bamHeaderArray[chromosome - 1]])
                     .enter().append('g')
-                    .attr('class', 'chromosome')
+                    .attr('class', 'chromosome-span')
```

Comment:
This class name confused me, because `span` is an HTML element type, and this is an SVG `g` element.

Reply:
It's generally better not to put element names in class names. Try to think of the meaning of the element, ie what it's purpose is in the specific context, and name it accordingly. If someone inspects the page, the class names should help them understand what each element's purpose is.